src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -99,7 +100,8 @@ export class GlobalMap extends React.Component<Props, State> {
 
   map: Mapbox.MapView
   clusterEngine: Supercluster
-  featureCollection: any
+  showsGeoJSONFeatureCollection: MapGeoFeatureCollection
+  fairsGeoJSONFeatureCollection: MapGeoFeatureCollection
```

Comment:
There's a geocoding types library in our app, so if I have to faff more, I'll switch to those, that's why there's an `as any` coming up.